### PR TITLE
Docs refresh

### PR DESCRIPTION
**Pull request by patrick-rodgers**
Update to add docs for the new v8 file picker, removes docs for versions older than 7.2 (based on usage data these aren't used).

### CODE REVIEW

File: docs/controls/file-pickers/v8-schema.md [hunk 1]
```diff
@@ -0,0 +1,205 @@
...
+          * @example
+          *  '/personal/user_contoso_com/Documents/Attachments'
+          */
+         folder?: string;
+      };
+      /**
+      * Indicates that File Picker should start in the user's recent files.
+      */
+      recent?: {};
+      /**
```
Comment (ThomasMichon) [inline docs/controls/file-pickers/v8-schema.md:106]: Remove.

File: docs/controls/file-pickers/v8-schema.md [hunk 2]
```diff
@@ -0,0 +1,205 @@
...
+      filters?: string[];
+
+      /**
+       * Configures whether or not specific pivots may be browsed for content by the user.
+       */
+      pivots?: {
+         recent?: boolean;
+         oneDrive?: boolean;
+         sharedLibraries?: boolean;
+         shared?: boolean;
```
Comment (ThomasMichon) [inline docs/controls/file-pickers/v8-schema.md:148]: Remove `search` and `shared` from `pivots`.

File: docs/controls/file-pickers/v8-schema.md [hunk 3]
```diff
@@ -0,0 +1,205 @@
...
+   };
+   /**
+    * Specified how many items may be picked.
+    */
+   selection?: {
+      /**
+       * @default 'single'
+       */
+      mode?: "single" | "multiple" | "pick";
+      /**
```
Comment (ThomasMichon) [inline docs/controls/file-pickers/v8-schema.md:160]: Remove `maxCount`

File: docs/controls/file-pickers/v8-schema.md [hunk 4]
```diff
@@ -0,0 +1,205 @@
...
+    */
+   commands?: {
+      /**
+       * Sets the default 'pick' behavior once the user selects items.
+       */
+      pick?: {
+         /**
+          * Does not currently have any effect, actions should be handled by the host application
+          */ 
+         action: "select" | "share" | "download" | "move";
```
Comment (ThomasMichon) [inline docs/controls/file-pickers/v8-schema.md:176]: Remove `action`.

File: docs/controls/file-pickers/v8-schema.md [hunk 5]
```diff
@@ -0,0 +1,205 @@
...
+         * The default is 'Cancel'.
+         * This string must be localized if provided.
+         */
+         label?: string;
+      };
+   };
+   /**
+    * Specifies accessibility cues such as auto-focus behaviors.
+    */
+   accessibility?: {
```
Comment (ThomasMichon) [inline docs/controls/file-pickers/v8-schema.md:196]: `accessibility` should be:
```ts
accessibility: {
  enableFocusTrap?: boolean;
  trapFocusOnLoad?: boolean;
  showFocusOnLoad?: boolean;
}
```

File: docs/controls/file-pickers/v8-schema.md [hunk 6]
```diff
@@ -0,0 +1,205 @@
...
+      focusTrap?: "initial" | "always" | "none";
+   };
+
+   /**
```
Comment (ThomasMichon) [inline docs/controls/file-pickers/v8-schema.md:200]: Remove `navigation`

File: docs/controls/file-pickers/v8-schema.md [hunk 7]
```diff
@@ -0,0 +1,205 @@
...
+             * @example
+             *  'General'
+             * @example
+             *  'foo/bar'
+             * @example
+             *  '/path/to/web/Shared Documents/General'
+             */
+            folder?: string;
+         };
+         /**
```
Comment (ThomasMichon) [inline docs/controls/file-pickers/v8-schema.md:74]: Remove `byId`.

File: docs/controls/file-pickers/v8-schema.md [hunk 8]
```diff
@@ -0,0 +1,168 @@
...
+          * Path segment for sub-folder within the user's OneDrive.
+          * @example
+          *  'Pictures'
+          * @example
+          *  '/personal/user_contoso_com/Documents/Attachments'
+          */
+         folder?: string;
+      };
+      /**
```
Comment (ThomasMichon) [inline docs/controls/file-pickers/v8-schema.md:92]: `recent` is under the `oneDrive` node.

File: docs/controls/file-pickers/v8-schema.md [hunk 9]
```diff
@@ -0,0 +1,168 @@
...
+         * Specifies that File Picker should start in the user's Files tab.
+         */
+         files?: {
+         /**
+          * Path segment for sub-folder within the user's OneDrive.
+          * @example
+          *  'Pictures'
+          * @example
+          *  '/personal/user_contoso_com/Documents/Attachments'
+          */
```
Comment (ThomasMichon) [inline docs/controls/file-pickers/v8-schema.md:89]: `folder` is under the `files` node.

**Review by ThomasMichon — APPROVED** [review]
A couple more nits in the `entry` config, but otherwise good to go.

**Comment by patrick-rodgers** [comment]
@ThomasMichon - thanks for the review - cleaned up the latest two items you noted. Ready for a final review. Thanks!

**Review by chackman — APPROVED** [review]

File: docs/controls/file-pickers/index.md [hunk 10]
```diff
@@ -1,41 +1,234 @@
...
+
+To initate the picker you need to create a "window" which can either be an iframe or a popup. Once you have a window you should construct a form and POST the form to the URL `{baseUrl}/_layouts/15/FilePicker.aspx` with the query string parameters defined.
+
+The `{baseUrl}` value above is either the SharePoint web url of the target web, or the user's onedrive. Some examples are: "https://tenant.sharepoint.com/sites/dev" or "https://tenant-my.sharepoint.com".
+
+- View [file picker configuration schema](./v8-schema.md).
+
+```TypeScript
+// create a new window
+const win = window.open("", "Picker", "width=800,height=600");
```
Comment (ThomasMichon) [inline docs/controls/file-pickers/index.md:58]: The recommended size for the Picker is 1080 x 680.

File: docs/controls/file-pickers/index.md [hunk 11]
```diff
@@ -1,41 +1,234 @@
...
+   1. Add `Files.Read.All`, `Sites.Read.All`, Leave `User.Read` for Graph delegated permissions
+   2. Add `AllSites.Read`, `MyFiles.Read` for SharePoint delegated permissions
 
+> If you are developing in [SharePoint Framework](https://aka.ms/spfx) you can [request these permissions](https://docs.microsoft.com/sharepoint/dev/spfx/use-aadhttpclient#request-permissions-to-an-azure-ad-application) in the application manifest with the resource "SharePoint" and "Microsoft Graph".
 
+## How it works
+
+To use the control you must:
+
+1. Make a POST request to the "control" page hosted at /_layouts/15/FilePicker.aspx. Using this request you supply some parameters, the key one being [the picker configuration](./v8-schema.md).
```
Comment (ThomasMichon) [inline docs/controls/file-pickers/index.md:39]: add " for any SharePoint site."

**Pull request merged**